README.md

```diff
@@ -103,4 +107,6 @@ This project has acceptance, integration and unit tests located inside the `test
 
 ## License
 
-This project is currently licensed under the Apache License version 2.0. A copy of `LICENSE` should be present along with the source code. To obtain the software under a different license, please contact [FOSSASIA](http://blog.fossasia.org/contact/).
+**This project is currently licensed under the [Apache License version 2.0](LICENSE).**
```

Comment:
Isn't this the incorrect syntax?

Reply:
I don't think so @sudheesh001 !
All the other `open-event-*` repo's are following the later one :wink:

Reply:
Looks like you tagged the wrong person. @0x48piraj you probably meant to tag @sumedh123